### PR TITLE
Bugfix: response might be empty (e.g. when blocked by AdBlocker)

### PR DESCRIPTION
**Pull request by plepe**
See plepe/ptmap#3 for details.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ ipLocation.Promise = typeof Promise === 'function' ? Promise : null // maybe you
 function _ipLocation (hostnameOrIP, callback) {
   ipLocation.httpGet('https://freegeoip.net/json/' + hostnameOrIP, function (err, res) {
     if (err) return callback(err)
+    if (!res) return callback('empty response')
```
Comment (jprichardson) [inline index.js]: Great idea. Please change this to `new Error('empty response')`.

**Comment by plepe** [comment]
done.

**Comment by jprichardson** [comment]
Published 1.0.1. Sorry for the long delay!!

**Comment by plepe** [comment]
Thanks!

**Pull request merged**